packages/datagrid/src/components/DefaultCellRenderer/QualityIndicator.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import { QUALITY_INVALID_KEY } from '../../constants';
+
+import theme from './QualityIndicator.scss';
+import { Quality } from '../../types';
+
+export interface QualityIndicatorProps {
+	qualityIndex: keyof Quality;
+}
+
+function QualityIndicator({ qualityIndex }: QualityIndicatorProps): JSX.Element {
+	const { t } = useTranslation();
+
+	if (qualityIndex !== QUALITY_INVALID_KEY) {
+		return <></>;
```

Comment:
Why not returning `null` ? To simplify return type ?

Reply:
siwtched to null, being lazy is not an excuse

Reply:
Brought tears to my eye